test/check-application

```diff
@@ -1569,6 +1569,95 @@ class TestFiles(testlib.MachineCase):
         b.click("li[data-location='/home/admin'] a")
         self.assert_last_breadcrumb("admin")
 
+        # Edit multiple files
+        create_multiple_dirs_sh = """
+            mkdir /home/admin/multiple
+            mkdir /home/admin/multiple/testdir
+        """
+        for i in range(20):
+            create_multiple_dirs_sh += f"touch /home/admin/multiple/filename{i}\n"
+
+        self.write_file("/usr/local/bin/create-multiple-dirs.sh", create_multiple_dirs_sh, perm="755")
+        m.execute("runuser -u admin /usr/local/bin/create-multiple-dirs.sh")
```

Comment:
Not that important yet, but in bootc /usr/local/ is read-only. Thus better do this from /tmp or `self.vm_tmpdir`